src/Reflections/HigherOrderCollectionMethodReflection.php

```diff
@@ -80,7 +81,7 @@ public function getName(): string
         return $this->reflections[0]->getName();
     }
 
-    public function getPrototype(): self
+    public function getPrototype(): ClassMemberReflection
```

Comment:
Thanks for sending this in, but this package enforces use of PHP >= 7.4, which handles this case correctly.

https://wiki.php.net/rfc/covariant-returns-and-contravariant-parameters
https://www.php.net/manual/en/language.oop5.variance.php

Reply:
Okay. I understand it is a parent class.